index.html

```diff
@@ -636,6 +636,526 @@ <h3>Criteria and Criterion</h3>
+        Additionally, criteria MAY have the following additional subcomponents
+          <ol type="a">
+            <li>
+              Source
+            </li>
+          </ol>
+        </li>
+        </ol>
+      </section> <!-- criteria -->
+      <section>
+        <h4>Criteria.Name</h4>
+        <p>Each criteria needs a human-friendly, short, descriptive name 
+        that captures the essence of the criteria as concisely as
+        possible.</p>
+      </section> <!-- Criteria.Name -->
+      <section>
+        <h4>Criteria.Id</h4>
+        <p>The criteria id must be explicit, unique, and persistent. See the section on Identifiers for details.
+</p>
+      </section> <!-- Criteria.Id -->
+      <section>
+        <h4>Criteria.Version</h4>
+        <p>The criteria version must increment, as appropriate, any time 
+        the criteria, or any of its subcomponents, is updated. See
+        the section on <a href="#versioning">Versioning</a> for details.</p>
+      </section> <!-- Criteria.Version -->
+      <section>
+        <h4>Criteria.Question</h4>
+        <p>This subcomponent presents the key question for evaluating the 
+        criteria. The question MUST be a single inquiry that gets to
+        the heart of the criteria. It should be reasonably clear to a 
+        competent professional skilled in the art how to resolve
+        the question into one of the possible responses.</p>
+      </section> <!-- Criteria.Question -->
+      <section>
+        <h4>Criteria.Responses</h4>
+        <p>This subcomponent lists the expected responses to the question.</p>
+        <ol>
+        <li>
+        Responses are either labeled or open ended.
+        </li>
+        <li>
+        Labeled responses MUST include a label and a meaning for that 
+        label.
+        </li>
+        <li>
+        Labels for a response MUST start with “A” in each criteria and progress sequentially through the English alphabet.
+        </li>
+        <li>
+        Open ended responses specify how an evaluator should fill in the response.
+        </li>
+        </ol>
+      </section> <!-- Criteria.Responses -->
+      <section>
+        <h4>Criteria.Relevance</h4>
+        <p>This subcomponent explains why this criteria is useful. Readers 
+        should be able to understand the extent to which this
+        particular criteria is applicable to their situation. 
+        </p>
+      </section> <!-- Criteria.Relevance -->
+      <section>
+        <h4>Criteria.Examples</h4>
+        <p>This section lists example evaluations of different DID methods 
+        using this criteria, presented as a table for easy
+        correlation between evaluations of different methods using the 
+        same criteria.
+        </p>
+        <ol>
+        <li>Each example must have the following entries, each its own column(s) in the examples table.
+        <ol type="a">
+          <li>Method</li>
+          <li>Responses (on column for each element)</li>
```

Comment:
```suggestion
          <li>Responses (one column for each labeled or open-ended response)</li>
```

Reply:
The "element[s]" in this case, are "Specification", "Network", and "Registry", which are not applicable to all criteria. Some only used one heading. This concept is underspecified, so I tried "element" as a generic, but it could have been clearer.

Reply:
Queue up in a new issue #58